src/arch/xtensa/boot_loader.c

```diff
@@ -218,15 +218,25 @@ static uint32_t hp_sram_init(void)
 
 #endif
 
-#if defined(CONFIG_ICELAKE)
+#if defined(CONFIG_APOLLOLAKE)
```

Comment:
Why are we removing support for ICL? Now all of the platforms have PLATFORM_HEAP_BUFFER = 3 and support for LP HEAP, but only for APL we are actually enabling LP SRAM. This doesn't seem right.

Reply:
Well no, PLATFORM_HEAP_BUFFER was changed only in `src/platform/apollolake/include/platform/memory.h` and will therefore apply only for apollolake.
As for why LPSRAM on ICL was disabled - we don't use LPSRAM on that platform at all so why would we keep that memory powered on? In fact it was me who enabled it about 3 months ago, but at that time we planed to use it on this platform, since plans have changed I decided to turn that off.

Reply:
@mrajwa the LPSRAM/HPSRAM for all platforms on/off and number of banks will be set via kconfig now, there is a pending PR.

Reply:
@lgirdwood, Yes I know, can we merge this PR now?

Reply:
@mrajwa will merge as soon as @tlauda approves latest updates

Reply:
@mrajwa @lgirdwood Approving, as only APL needs manual LPSRAM enablement. For rest of the cAVS platforms ROM enables the LPSRAM.